packages/caliper-ethereum/lib/ethereum-connector.js

```diff
@@ -402,8 +402,8 @@ class EthereumConnector extends ConnectorBase {
             if (txRcpt.status === '0x1') {
                 return new web3.eth.Contract(contractData.abi, txRcpt.contractAddress);
             } else {
-                logger.error('Failed private transaction hash ' + txHash);
-                throw new Error('Failed private transaction hash ' + txHash);
+                logger.error(`Failed private transaction hash ${txHash}`);
```

Comment:
As this message is duplicated, better to set it as a variable and pass it into `logger.error` and `new Error`